### PR TITLE
Added option to send updates to attendees

### PR DESCRIPTION
**Pull request by kenomaerz**
As response to issue #50. Adds option to send updates to attendees for all methods supporting sendUpdates and currently implemented in gcsa.  

Open points for discussion:
1. Location of the class encapsulating the values: Currently at the top of google_calendar.py
2. Default value: Currently "None". No explicit default given in docs, but google behaves that way currently apparently.
3. No tests added yet, as I couldn't identify a matching testing infrastructure

### CODE REVIEW

File: gcsa/google_calendar.py [hunk 1]
```diff
@@ -79,7 +92,7 @@ def _get_credentials(self):
 
         return credentials
 
-    def add_event(self, event):
+    def add_event(self, event, sendUpdates=SendUpdatesMode.NONE):
```
Comment (kuzmoyev) [inline gcsa/google_calendar.py:95]: Trying to pep8-ify the library, so I'd use snake case here (`send_updates`)

**Comment by kuzmoyev** [comment]
Thank you for the PR. Very appreciated. One small change is required. Apart from that, looks perfect. Will merge it and add appropriate documentation, when I have time.

**Comment by kenomaerz** [comment]
Adjusted to use snake case, which it really should have been initially :)

File: gcsa/google_calendar.py (hunk 1 above)
Comment (kenomaerz) [reply gcsa/google_calendar.py]: Done!

**Pull request merged**